src/cargo/ops/registry/mod.rs

```diff
@@ -116,13 +118,12 @@ impl RegistryCredentialConfig {
 /// * `token_required`: If `true`, the token will be set.
 fn registry(
     gctx: &GlobalContext,
+    source_ids: &RegistrySourceIds,
```

Comment:
Should we split this into a separate PR?  At least when I did command merges, I tried to keep the merge-PR minimal.

Reply:
Ok, I will split this into a separate PR.